pandas/tests/test_format.py

```diff
@@ -2952,6 +2952,16 @@ def test_to_csv_date_format(self):
         self.assertEqual(df_day.to_csv(), expected_default_day)
         self.assertEqual(df_day.to_csv(date_format='%Y-%m-%d'), expected_default_day)
 
+        # GH 7791
```

Comment:
well, its not implemented ATM, so pls create another issue (its pretty easy to fix by basically copying the logic from ``core/internals/FloatBlock.to_native_types`` where it is implemented). But let's do that in a separate issue. This PR should go first.